core/src/test/java/hudson/tasks/maven/MavenTest.java

```diff
@@ -0,0 +1,30 @@
+package hudson.tasks.maven;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import hudson.tasks.Maven;
+
+import org.junit.Test;
+
```

Comment:
Hmmm, sorry, I really don't know how the tests slipped out, they aren't ready yet. I did only push to my copy/clone...
I couldn't build the project at work, because the proxy/firewall blocks `com.sun.winsw:winsw:exe:bin:1.16`, so I had to restart over on my private machine (toolchain, etc) but it got late last night.